README.rst

```diff
@@ -1,6 +1,13 @@
+.. image:: docs/images/robocop_logo.png
+   :scale: 50 %
```

Comment:
Either scaling does not work or the logo is still too big. I would make it smaller :)

Reply:
I've tested scaling with readthedocs, not readme (where I moved logo later). So scaling works but like you wrote I need to make it smaller